amplify/backend/function/PropertyFunction/src/index.py

```diff
@@ -177,7 +229,7 @@ def handle_get_request(event):
 
     
 
-def handle_delete_request(event):
+def handle_delete_request(event,db):
```

Comment:
Shouldn't we be soft-deleting here (i.e. changing status to zero), instead of deleting the record?